### DISCUSSION TITLE
creating a route walking paths

### DISCUSSION THREAD

**Discussion by buildvoc10** [acteng/atip#123] [Q&A]
can your a route be created on walking paths/trail ? https://buildingshistory.co.uk/#16.89/51.205542/-0.743767 see an example of a gpx route

**Comment by dabreegster** [comment]
Yes. If it's not working, please let me know -- it's possible some paths currently are filtered out from the import. You can try at https://acteng.github.io/atip/. And the route drawing tool can be incorporated in your own project as an NPM package; see https://github.com/dabreegster/route_snapper/

**Comment by buildvoc10** [comment]
Thanks was able to export a path this morning to geojson, looking for a way to edit OS 3D buildings in the browser any ideas?

**Comment by dabreegster** [comment]
OS as in Ordnance Survey? No idea. For OSM / OpenStreetMap, maybe something at https://wiki.openstreetmap.org/wiki/3D

I'm going to convert this into a discussion, to keep the issue tracker open for things related to ATIP. Feel free to keep asking questions / giving feedback though!

**Reply by buildvoc10** [reply]
@dabreegster  using the app over the weekend for hiking trail [](https://buildingshistory.co.uk/#15.04/51.2463/-0.82239) ![image](https://user-images.githubusercontent.com/76884997/232599194-6c5b168d-e12b-4a50-9fbc-9b59ffc9bd2a.png) I like the way it caches your transport selection i.e Surrey Hampshire 😄 Phil
